thoth/metrics_exporter/jobs/pi.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# thoth-metrics
+# Copyright(C) 2018, 2019 Christoph Görn, Francesco Murdaca, Fridolin Pokorny
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Performance indicator metrics."""
+
+import logging
+
+from thoth.storages import GraphDatabase
+import thoth.metrics_exporter.metrics as metrics
+
+from .base import register_metric_job
+from .base import MetricsBase
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class PIMetrics(MetricsBase):
+    """Class to discover Content for Performance Indicators inside Thoth database."""
+
+    _ML_FRAMEWORKS = ["tensorflow"]
```

Comment:
It would be a good idea to create a query in thoth-storages to get listing of all the available frameworks and iterate over them rather than maintaining this list explicitly.

Reply:
https://github.com/thoth-station/storages/pull/1180 here you have it :beers: :)